packages/components/src/typography/typography.stories.tsx

```diff
@@ -7,45 +7,45 @@ export default {
   component: Typography,
 };
 
-export const SizeRamp = (): JSX.Element => (
+export const LegacySizeRamp = (): JSX.Element => (
```

Comment:
Is the idea that existing usages in Learning Platform can use the legacy ramp, and then be migrated to non-legacy over time?

Reply:
yeah exactly, it'll make this first migration easier. then there are a couple directions the non-legacy sizing can go, but we can figure it out later